mcu_program/common_src/lms7002m_calibrations.c

```diff
@@ -84,13 +84,15 @@ void LoadDC_REG_TX_IQ()
     FlipRisingEdge(TSGDCLDQ_TXTSP);
 }
 
-static void Dummy()
+#if 0 // nobody needs this right now.
```

Comment:
I think it is used when compiling calibrations image for 8051 MCU. Also, I think at least some of the warnings in C files located in 'mcu_program' folder might be intentional and meant to deal with memory/performance/timing constraints in MCU.